example/src/investments.ts

```diff
@@ -0,0 +1,30 @@
+import dotenv from 'dotenv'
+import Pluggy from 'pluggy-js'
+import { sleep, PLUGGY_BANK_CREDENTIALS, PLUGGY_BANK_CONNECTOR } from './utils'
+
+dotenv.config()
+
+const { PLUGGY_API_KEY: pluggyApiKey, PLUGGY_API_URL } = process.env
+const baseUrl = PLUGGY_API_URL || undefined
```

Comment:
It would remove things like this from examples.

Reply:
updated